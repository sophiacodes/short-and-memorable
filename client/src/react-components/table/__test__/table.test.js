import React from 'react';
import Table from '../';
import renderer from 'react-test-renderer';

it('renders <Table /> correctly', () => {
  const header = [
      'Header 1',
      'Header 2'
  ];
  const data = {
    'Key 1': 'Item 1',
    'Key 2': 'Item 2'
  };
  const caption = 'Caption this!';
  const footer = 'This is a footer';
  const tree = renderer
    .create(<Table 
        headers={header}
        data={data}
        caption={caption}
        footer={footer}
    />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});