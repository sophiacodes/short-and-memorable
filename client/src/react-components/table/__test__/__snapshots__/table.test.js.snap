// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders <Table /> correctly 1`] = `
<table
  className="table"
>
  <caption>
    Caption this!
  </caption>
  <thead>
    <tr>
      <th>
        Header 1
      </th>
      <th>
        Header 2
      </th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>
        Key 1
      </td>
      <td>
        Item 1
      </td>
    </tr>
    <tr>
      <td>
        Key 2
      </td>
      <td>
        Item 2
      </td>
    </tr>
  </tbody>
  <tfoot>
    <tr>
      <td
        colSpan={2}
      >
        This is a footer
      </td>
    </tr>
  </tfoot>
</table>
`;
